.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Checklist for this pull request
+Before submitting a pull request, please make sure you have to following:
+
+* Your code should contain tests relevant for the problem you are solving
+* Your commits messages format follows the jQuery git commit message format (http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines)
```

Comment:
"Should follow"

Since I know how to rework commits etc it is no longer a hard dependency